TACC/Core-Portal
# Task/fp 341  ui fixes

## Overview: ##

Fix UI issues—for #87—from browser bugs, layout update vs AutoSizer, and code deleted in 6d757ff.

_**Update**: Also includes fixes to "routing and tests" (317e4ed)._

## PR Status: ##


## Related Jira tickets: ##

* [FP-341](https://jira.tacc.utexas.edu/browse/FP-341)

## Summary of Changes: ##

- **Avoid Edge/Chrome browser bug**\
    (a `<fieldset>` can not be `display: flex`)
    - _accidentally already committed directly to parent branch: ebbc7e1_
- **Restore deleted `className`** (used, but deleted in 6d757ff)
    - _accidentally already committed directly to parent branch: c02dbc9_
- **Restore `margin: auto`**\
    (element matching selector deleted in 6d757ff, so important style needs new selector)
- **Remove wrap div that was needed for `<AutoSizer>`**\
    (before layout change from `task/FP-341--ui-ux`)

## Testing Steps: ##
1. Ensure UI matches screenshots, in all browsers.
1. Ensure table loads and scrolls, in all browsers—especially Safari.

## UI Photos:

### Simple UI Fixes
- Chrome\
    <img width="851" alt="FP-341 Chrome" src="https://user-images.githubusercontent.com/62723358/89587193-bcf81600-d806-11ea-8c31-cc5989e0096b.png">
- Edge\
    <img width="851" alt="FP-341 Edge" src="https://user-images.githubusercontent.com/62723358/89587194-bd90ac80-d806-11ea-8881-b7ed146c4128.png">
- Firefox\
    <img width="851" alt="FP-341 Firefox" src="https://user-images.githubusercontent.com/62723358/89587196-bd90ac80-d806-11ea-96a7-8b0036112918.png">
- Safari\
    <img width="851" alt="FP-341 Safari" src="https://user-images.githubusercontent.com/62723358/89587198-be294300-d806-11ea-9317-1f03bb1212c1.png">

### Missing Table Content in Safari

- Safari\
    <img width="1280" alt="FP-341 Safari Table - Now Fixed" src="https://user-images.githubusercontent.com/62723358/89588628-a606f300-d809-11ea-8f88-1fa8ea3f640f.png">
- Firefox\
    <img width="1280" alt="FP-341 Firefox Table - Still Works" src="https://user-images.githubusercontent.com/62723358/89588636-a8694d00-d809-11ea-95d5-187aeb00cd63.png">

## Notes: ##
